test/errors-spec.js

```diff
@@ -293,22 +296,22 @@ describe("Lexer:", function() {
         it("reject 3-digit hex notation without #", function() {
             expect`\textcolor{1a2}{foo}`.toFailWithParseError(
                    "Invalid color: '1a2'" +
-                   " at position 12: \\textcolor{1̲a̲2̲}{foo}");
+                   " at position 11: \\textcolor{̲1̲a̲2̲}̲{foo}");
```

Comment:
Why is there an extra space inside of `\textcolor{}` in the error message?

Reply:
@kevinbarabash It is probably combining underline not shown properly.

Reply:
I don't see an extra space anymore. 🤷‍♂️